queries.sql

```diff
@@ -8,3 +8,68 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name NOT IN ('Gabumon');
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <=17.3;
+
+
+BEGIN;
+
+UPDATE animals
+SET species = 'unspecified';
+
+ROLLBACK;
+
+SELECT * FROM animals;
+
+
+
+BEGIN;
+
+UPDATE animals SET species = 'digimon' WHERE name like '%mon';
+
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+
+COMMIT;
+
+SELECT * FROM animals;
+
+
+BEGIN;
+
+DELETE FROM animals;
+
+ROLLBACK;
+
+SELECT * FROM animals;
+
+
+BEGIN;
+
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+
+SAVEPOINT Point1;
+
+UPDATE animals SET weight_kg = weight_kg * -1;
+
+ROLLBACK TO Point1;
+
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+
+COMMIT;
```

Comment:
- [ ] Well! As I mentioned previously, because you did not insert the right data, your current query is returning the wrong output. Please consider updating your inserted data and add an updated screenshot for this query. Your query should reflect the accurate data as per the project requirements to ensure the correctness of your project. Keep up the good work and attention to detail! 👏📊🐾